src/Gizmos/gizmo.ts

```diff
@@ -166,6 +166,14 @@ export class Gizmo implements IDisposable {
         });
     }
 
+    /**
+     * update customRotationQuaternion
+     * @param customRotationQuaternion Quaternion that defines the gizmo‘s pose
+     */
+    public setCustomRotationQuaternion(customRotationQuaternion: Nullable<Quaternion>) {

```

Comment:
Why not using standard getter / setter?

```typescript
public get customRotationQuaternion():  Nullable<Quaternion>) {
    return this._customRotationQuaternion;
}

public set customRotationQuaternion(customRotationQuaternion: Nullable<Quaternion>) {
    this._customRotationQuaternion = customRotationQuaternion;
}

Reply:
that could be better!

and,I wonder if it's necessary to open the getter method. Are we going to use the getter?

Reply:
I don't know, but even if it's not, it is a one liner so that's not a big deal and if someone ever wants it one day it is there!